verification/src/tracer.rs

```diff
@@ -0,0 +1,24 @@
+use opentelemetry::{sdk::trace::Tracer, trace::TraceError};
+use tracing_subscriber::{filter::LevelFilter, layer::SubscriberExt, prelude::*};
+
+pub fn init_logs() {
```

Comment:
May be add "enabled" option, so people could disable tracing altogether (or even disable them by default)
but leave the logs

Reply:
what do you mean by `disable tracing`? we use tracing for logs. you mean disable jaeger?